raster-foundry/raster-foundry
# Add migration to add delete cascade to user FKs

## Overview

This PR adds a migration that drops and re-adds all user table FKs with `ON DELETE CASCADE`.

There were some `NOT VALID` FKs and those were dropped but not re-added.

### Checklist


## Testing Instructions

- In psql, `\d users` and notice that no constraints have `ON DELETE CASCADE`
- Run migrations
- in psql, `\d users` and notice that all constraints have `ON DELETE CASCADE`
